frontend/src/components/professor/ActivityAssessmentList/ActivityAssessmentList.js

```diff
@@ -1,56 +1,57 @@
 import { Content } from '../../App/AppGeneralStyles'
 import { Col } from 'react-bootstrap'
 import ActivityListItem from './ActivityListItem'
+import { useEffect, useState } from 'react'
+import ProfessorService from '../../../services/professor.service'
+import ExpeditionService from '../../../services/expedition.service'
 
-// todo: get from endpoint, remove
-const exampleActivities = {
-  activites: [
-    {
-      type: 'task',
-      source: 'Rozdział 1',
-      name: 'Supersystem',
-      remaining: 33,
-      activityId: 0
-    },
-    {
-      type: 'task',
-      source: 'Rozdział 1',
-      name: 'Supersystem II',
-      remaining: 23,
-      activityId: 1
-    },
-    {
-      type: 'task',
-      source: 'Rozdział 1',
-      name: 'Supersystem III',
-      remaining: 13,
-      activityId: 2
-    },
-    {
-      type: 'task',
-      source: 'Rozdział 1',
-      name: 'Supersystem IV',
-      remaining: 13,
-      activityId: 3
-    },
-    {
-      type: 'task',
-      source: 'Rozdział 1',
-      name: 'Supersystem V',
-      remaining: 2,
-      activityId: 4
-    }
-  ]
-}
+// note: currently the list assumes we only can only manually grade File Tasks - this is due to the way our DB currently works,
+// an ID is unique only in the task group. we might need to add task type differentation on the backend if we want to check
+// other types of activities in the future
 
 export default function ActivityAssessmentList() {
+  const [activityList, setActivityList] = useState('')
```

Comment:
Nie wiem czy to dobry pomysł, żeby wartość domyślna to był pusty string. Ja bym raczej dał undefined